dali/sasha/sasha.cpp

```diff
@@ -405,10 +405,11 @@ int main(int argc, char* argv[])
 
     EnableSEHtoExceptionMapping();
     Thread::setDefaultStackSize(0x10000);
-    startMPServer(0);
-    attachStandardFileLogMsgMonitor("sasha.log", NULL, MSGFIELD_STANDARD, MSGAUD_all, MSGCLS_all, TopDetail, false, true);
-    queryStderrLogMsgHandler()->setMessageFields(MSGFIELD_prefix);
     try {
+        startMPServer(0);
+        attachStandardFileLogMsgMonitor("sasha.log", NULL, MSGFIELD_STANDARD, MSGAUD_all, MSGCLS_all, TopDetail, false, true);
+        queryStderrLogMsgHandler()->setMessageFields(MSGFIELD_prefix);
```

Comment:
NB: moving this inside the try is an incidental change.

It means if an exception is thrown here it will now be caught and reported properly - previously the exception in attachStandardFileLogMsgMonitor was unhandled.